user_input/osc_input.py

```diff
@@ -50,8 +52,8 @@ def setup_osc_server(self):
         this_dispatcher.map("/shaderparam2", self.on_param_osc_input)
         this_dispatcher.map("/shaderparam3", self.on_param_osc_input)
         this_dispatcher.map("/shutdown", self.exit_osc_server)
-        #this_dispatcher.map("/*", print)        
-
+        this_dispatcher.map("/*", self.on_param_osc_input)
```

Comment:
i think there is a problem , that the /* path will overwrite the paths defined here - for example when /shutdown is run it will still call the on_param+osc_input method - maybe ther is a way to set the prioty , but an easier way might be to have all functions go through the on_param_osc_input method . i will try this today